Makefile

```diff
@@ -0,0 +1,51 @@
+ifndef VERSION
+	VERSION = ${shell git rev-parse --abbrev-ref HEAD}-${shell git rev-parse --short HEAD}
+
+endif
+
+ifeq (,$(shell go env GOBIN))
+	GOBIN=$(shell go env GOPATH)/bin
+else
+	GOBIN=$(shell go env GOBIN)
+endif
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+GOLANG_CI_LINT = $(LOCALBIN)/golangci-lint
+GOLANG_CI_LINT_VERSION ?= v1.54.2
+
+.PHONY: lint
+lint:
+	GOBIN=$(LOCALBIN) go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANG_CI_LINT_VERSION)
+	$(LOCALBIN)/golangci-lint run -v
+
+FLAGS = -ldflags '-s -w -X github.com/kyma-project/cli/cmd/kyma/version.Version=$(VERSION)'
+
+.PHONY: build
+build: build-windows build-linux build-darwin build-windows-arm build-linux-arm build-darwin-arm
```

Comment:
The build emits binaries named: `kyma-linux` etc. Please change that to `modulectl-<OS>`